README.md

```diff
@@ -87,6 +87,12 @@ The Ginkgo CUDA module has the following __additional__ requirements:
 The Ginkgo OMP module has the following __additional__ requirements:
 *  _MinGW_ or _Cygwin_
 
+Need to add the following manually:
+* Test Ginkgo with shared Ginkgo library:
+  Add `PROJECT_BINARY_DIR/GINKGO_WINDOWS_SHARED_LIBRARY_RELPATH` into the environment `PATH`. `GINKGO_WINDOWS_SHARED_LIBRARY_RELPATH` is `windows_shared_library` by default. More Details are availble in the [Installation page](./INSTALL.md).
```

Comment:
wording suggestion/typos:
```suggestion
  Add `PROJECT_BINARY_DIR/GINKGO_WINDOWS_SHARED_LIBRARY_RELPATH` to the `PATH`  environment variable. `GINKGO_WINDOWS_SHARED_LIBRARY_RELPATH` is `windows_shared_library` by default. More Details are available in the [Installation page](./INSTALL.md).
```

Reply:
I keep "into the environement PATH" to be consistent with other documents in ginkgo